--- a/Content/Product/DataAbstractionLayer/StockUpdater.php
+++ b/Content/Product/DataAbstractionLayer/StockUpdater.php
@@ -316,13 +316,12 @@
 
         $update = new RetryableQuery(
             $this->connection,
-            $this->connection->prepare('UPDATE product SET available_stock = stock - :open_quantity, sales = :sales_quantity, updated_at = :now WHERE id = :id')
+            $this->connection->prepare('UPDATE product SET available_stock = stock , sales = :sales_quantity, updated_at = :now WHERE id = :id')
         );
 
         foreach ($fallback as $id) {
             $update->execute([
                 'id' => Uuid::fromHexToBytes((string) $id),
-                'open_quantity' => 0,
                 'sales_quantity' => 0,
                 'now' => (new \DateTime())->format(Defaults::STORAGE_DATE_TIME_FORMAT),
             ]);
@@ -331,7 +330,6 @@
         foreach ($rows as $row) {
             $update->execute([
                 'id' => Uuid::fromHexToBytes($row['product_id']),
-                'open_quantity' => $row['open_quantity'],
                 'sales_quantity' => $row['sales_quantity'],
                 'now' => (new \DateTime())->format(Defaults::STORAGE_DATE_TIME_FORMAT),
             ]);
